scrapy/pipelines/images.py

```diff
@@ -177,7 +185,16 @@ def _warn():
         ## end of deprecation warning block
 
         image_guid = hashlib.sha1(to_bytes(url)).hexdigest()  # change to request.url after deprecation
-        return 'full/%s.jpg' % (image_guid)
+        ext = "jpeg"
```

Comment:
jpeg and jpg are interchangeable right? I kept it as jpeg to be consistent with the PIL format. ( when I do `ext = Image.format.lower()` )